bin/validate.py

```diff
@@ -7,8 +7,10 @@
 import zipfile
 
 if __name__ == "__main__":
```

Comment:
If we are shifting to using containers for validation and scoring, are this script and the `model_to_data_score.py` script still needed in this repo? Potential opportunity to eliminate some code that we need to maintain.

Reply:
It looks like `DATA_TO_MODEL` still uses [the validation script](https://github.com/Sage-Bionetworks-Workflows/nf-synapse-challenge/blob/dffa9497f1eb6316f3acdfcd5d19e52638967021/workflows/DATA_TO_MODEL.nf#L11). I can remove the `model2data` scoring script, and leave the validation in until `data2model` is ready to switch to containers.

Reply:
Let's create a ticket so we don't lose track of this

Reply:
[ticket](https://sagebionetworks.jira.com/browse/ORCA-325)